e2e/framework/addon/eso.go

```diff
@@ -33,22 +33,10 @@ func NewESO(mutators ...MutationFunc) *ESO {
 			ReleaseName: "eso",
 			Chart:       "/k8s/deploy/charts/external-secrets",
 			Vars: []StringTuple{
-				{
-					Key:   "image.repository",
-					Value: os.Getenv("IMAGE_REGISTRY"),
-				},
```

Comment:
This is defaulted in the helm chart, there is no need to pass in the value. I removed it here and in all other places that relied on it.